### ISSUE TITLE
Customizable Rollups UI Error When Einstein Opportunity Insights is Enabled

### ISSUE THREAD

**Issue by jesslopez-sf** [SalesforceFoundation/NPSP#4022]
When Einstein Opportunity Insights is enabled, the Customizable Rollups UI returns an error when attempting to load an existing rollup or create a new one:
**_"Error displaying the Rollup
List index out of bounds:0"_**

![screen shot 2019-01-29 at 4 00 54 pm](https://user-images.githubusercontent.com/29731456/51940249-78a0e400-23df-11e9-8b41-8698b4567744.png)

The error occurs in [Line 479 of CRLP_RollupUI_SVC](https://github.com/SalesforceFoundation/Cumulus/blob/master/src/classes/CRLP_RollupUI_SVC.cls#L478-L479).
`fieldObject.put('referenceTo', dfr.referenceTo[0].getDescribe().name);`

Most fields of type Reference have a [0] referenceTo item, but this error was occurring when a field called "topinsightid" was encountered. This field is specific to Einstein Opportunity Insights. We aren't excluding it because it doesn't contain ___system_ in the name ([line 471](https://github.com/SalesforceFoundation/Cumulus/blob/master/src/classes/CRLP_RollupUI_SVC.cls#L471)).

![screen shot 2019-01-29 at 4 02 25 pm](https://user-images.githubusercontent.com/29731456/51940451-0d0b4680-23e0-11e9-947c-c348f892f896.png)

**Stack Trace:**
![screen shot 2019-01-29 at 4 02 50 pm](https://user-images.githubusercontent.com/29731456/51940482-2a401500-23e0-11e9-9280-aad14dc3cb87.png)

**Comment by jesslopez-sf** [comment]
**Lurch: Add

**Comment by coriobriensfdo** [comment]
Workaround:

1) From NPSP Settings > Customizable Rollups, I disabled the Customizable Rollups option. 
2) From Setup > Einstein > Einstein Sales > Assisted Setup, I disabled Einstein Opportunity Insights, and Einstein Opportunity Scoring. All other Insights I left enabled. 
3) From NPSP Settings > Customizable Rollups, I enabled the Customizable Rollups option. Thereafter, I was able to open existing rollups and create new rollups without the error.
